tests/plugins/test_cbsnews.py

```diff
@@ -0,0 +1,24 @@
+import unittest  # noqa: F401
+import pytest
+from streamlink.plugins.cbsnews import CBSNews
+
+
+class TestCBSNews:
```

Comment:
What is the reason for not using `unittest.TestCase`?
We shouldn't be mixing test setups. Almost all tests are written with unittest and pytest is only used as the test runner. Mixing this makes it much more difficult to read and maintain.

Reply:
This has been used previously; we have `pytest` available and it has superior test builder functions.